receiver/sqlserverreceiver/documentation.md

```diff
@@ -289,6 +313,100 @@ This metric is only available when the receiver is configured to directly connec
 | file_type | The type of file being monitored. | Any Str |
 | direction | The direction of flow of bytes or operations. | Str: ``read``, ``write`` |
 
+### sqlserver.database.tempdb.space
+
+Total free space in temporary DB.
+
+| Unit | Metric Type | Value Type | Aggregation Temporality | Monotonic |
+| ---- | ----------- | ---------- | ----------------------- | --------- |
+| “KB” | Sum | Int | Cumulative | false |
```

Comment:
👋 I don't know if it's intended but there is a few occurrences of `“` being used instead of the usual `"` in this PR